pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -20,16 +20,21 @@ details[open]>summary::before {
 
 > [!primary]
 >
-> VCD on OVHcloud is currently in alpha phase. This guide may therefore be incomplete and can be updated in the future. 
-> 
+> VCD on OVHcloud is currently in alpha phase. This guide may therefore be incomplete and can be updated in the future.
+>
 
 ## Objective
 
 **The aim of this guide is to audit the special cases of vSphere managed HPC that may pose problems for migration to VCD on OVHcloud.**
```

Comment:
@pg-ovh or @OVH-Team-Guides Do you have an idea of something less alarming than the word “problems"?

Reply:
"L’objectif est de fournir une liste des cas d’utilisation complexes et les solutions à mettre en place pour migrer vers VCD." ?